### PR TITLE
DOCKER-894 NFS shared volumes tests should check for "NFS shared volumes not supported" error in a separate tests suite'

### PR DESCRIPTION
**Pull request by joshwilsdon**

### CODE REVIEW

**Comment by joshwilsdon** [comment]
@misterdjules: please hold off on the review for now, I just found an issue which I will try to resolve first and then re-request review.

**Comment by misterdjules** [comment]
@joshwilsdon Sounds good! Just let me know when it's ready to review.

File: test/integration/api-mount-nfs-shared-volume-as-docker-compose.test.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright 2016, Joyent, Inc.
+ * Copyright 2017, Joyent, Inc.
```
Comment (misterdjules) [inline test/integration/api-mount-nfs-shared-volume-as-docker-compose.test.js]: Nice!

**Comment by joshwilsdon** [comment]
merged with c0a66afabc517511640c7887e55321da905fa99b

**Pull request closed without merging**